app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -324,6 +324,12 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi {
     finish()
```

Comment:
Was "blood" in the commit message meant to be "blood sugar"?

Reply:
Yes, reworded the commit message `Disable background click of sheet if blood sugar reading is not blank`